server/services/action.js

```diff
@@ -303,6 +303,51 @@ const actions = {
                                 'link': `${response.data.reference}`
                             };
 
+                            return handleActionSuccess(clientResponse, {}, form);
+                        }
+                        case actionTypes.RECORD_INTEREST: {
+                            const { caseActionData } = options;
+
+                            let requestBody = {
+                                actionType: 'RECORD_INTEREST',
+                                caseTypeActionUuid: caseActionData.EXTERNAL_INTEREST[0].typeInfo.uuid,
+                                caseTypeActionLabel: 'RECORD_INTEREST',
```

Comment:
I think... if you implement a strictness of using the source info in the `hocs-casework` you wouldn't need to populate this here... same as in appeals above. not a huge thing though if you want to leave it.